### PR TITLE
Feature/avatar upload

### PR DESCRIPTION
**Pull request by doxliydriw**
Avatar update implemented.
Buttons used were updated in accordance to design system.

### CODE REVIEW

File: src/routes/AccountRoute/AccountRouteStyled.js [hunk 1]
```diff
@@ -10,6 +10,9 @@ border-left: 1px solid var(--color-grey-6);
 @media (max-width: 767px) {
```
Comment (ReginaVarybrus) [inline src/routes/AccountRoute/AccountRouteStyled.js]: Let's use our mediaQuery.js from constants. It has to be like this `@media ${device.tablet} ...`

File: src/routes/AccountRoute/AccountRouteStyled.js [hunk 2]
```diff
@@ -51,40 +54,49 @@ box-sizing: border-box;
...
-export const ChangeAvatar = styled(Button)`
-padding-top: 8px;
-color: var(--color-brand-blue);
-width: 120px;
 
-@media (max-width: 767px) {
-    padding: 8px 16px;
-}
-`
+export const AvatarVisuallyHiddenInput = styled('input')({
```
Comment (ReginaVarybrus) [inline src/routes/AccountRoute/AccountRouteStyled.js]: Probably here has to be `... = styled.input'...' `?

**Review by ReginaVarybrus — CHANGES_REQUESTED** [review]

File: src/routes/AccountRoute/AccountRouteStyled.js (hunk 2 above)
Comment (doxliydriw) [reply src/routes/AccountRoute/AccountRouteStyled.js]: removed because was not needed.

File: src/routes/AccountRoute/AccountRouteStyled.js (hunk 1 above)
Comment (doxliydriw) [reply src/routes/AccountRoute/AccountRouteStyled.js]: updated accordingly.

**Pull request merged**